### PR TITLE
NMS-13294: Add Meridian repository description for Minion and Sentinel

### PR DESCRIPTION
**Pull request by indigo423**
### All Contributors

* [x] Have you read and followed our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [x] Have you [made an issue in the OpenNMS issue tracker](https://issues.opennms.org)?<br>If so, you should:
  1. update the title of this PR to be of the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the JIRA link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
* [x] Have you made a comment in that issue which points back to this PR?
* [x] Have you updated the JIRA link at the bottom of this comment to link to your issue?
* [x] If this is a new or updated feature, is there documentation for the new behavior?
* [x] If this is new code, are there unit and/or integration tests?
* [x] If this PR targets a `foundation-*` branch, does it avoid changing files in `$OPENNMS_HOME/etc/`?

Adding Meridian repository description for Minion and Sentinel.

* JIRA (Issue Tracker): http://issues.opennms.org/browse/NMS-13294

### CODE REVIEW

File: docs/modules/deployment/pages/minion/centos-rhel7/install-minion.adoc [hunk 1]
```diff
@@ -4,16 +4,36 @@
...
-endif::[]
 ----
+endif::[]
+
+////
+Repo: Meridian Minion
+////
 ifeval::["{page-component-title}" == "Meridian"]
-Configure the repository by copying the Meridian repository file you received from OpenNMS Sales to the Minion server.
+.Add repository and import GPG key
```
Comment (Bonrob2) [inline docs/modules/deployment/pages/minion/centos-rhel7/install-minion.adoc:23]:
```suggestion
. Add repository and import GPG key.
```

File: docs/modules/deployment/pages/minion/centos-rhel8/install-minion.adoc [hunk 2]
```diff
@@ -4,18 +4,37 @@
...
 ----
-ifeval::["{page-component-title}" == "Meridian"]
-Configure the repository by copying the Meridian repository file you received from OpenNMS Sales to the Minion server.
 endif::[]
 
+////
+Repo: Meridian Minion
+////
+ifeval::["{page-component-title}" == "Meridian"]
+.Add repository and import GPG key
```
Comment (Bonrob2) [inline docs/modules/deployment/pages/minion/centos-rhel8/install-minion.adoc:23]:
```suggestion
. Add repository and import GPG key.
```

**Review by Bonrob2 — CHANGES_REQUESTED** [review]
LGTM!

**Review by mmahacek — COMMENTED** [review]
Do we want to maintain duplicate repo instructions in Minion/Sentinel, or have those components just reference to use the same repo as the core server?  This would make future edits easier as there wouldn't be as many copies to have to keep in sync when.

**Comment by Bonrob2** [comment]
> Do we want to maintain duplicate repo instructions in Minion/Sentinel, or have those components just reference to use the same repo as the core server? This would make future edits easier as there wouldn't be as many copies to have to keep in sync when.

This makes sense to me.

**Comment by indigo423** [comment]
@mmahacek @Bonrob2 I have addressed your comment regarding duplication for the repository file.

**Review by mmahacek — APPROVED** [review]

**Review by Bonrob2 — APPROVED** [review]
LGTM!

**Pull request merged**